src/patternfly/components/Nav/nav.scss

```diff
@@ -219,64 +77,41 @@
     position: relative;
     display: flex;
     align-items: baseline;
-    padding:
-      var(--pf-c-nav__list-link--PaddingTop)
-      var(--pf-c-nav__list-link--PaddingRight)
-      var(--pf-c-nav__list-link--PaddingBottom)
-      var(--pf-c-nav__list-link--PaddingLeft);
+    padding: var(--pf-c-nav__list-link--PaddingY) var(--pf-c-nav__list-link--PaddingX);
     font-weight: var(--pf-c-nav__list-link--FontWeight);
     color: var(--pf-c-nav__list-link--Color);
 
     @media screen and (min-width: $pf-global--breakpoint--md) {
-      --pf-c-nav__list-link--PaddingRight: var(--pf-c-nav__list-link--md--PaddingRight);
-      --pf-c-nav__list-link--PaddingLeft: var(--pf-c-nav__list-link--md--PaddingLeft);
+      --pf-c-nav__list-link--PaddingX: var(--pf-c-nav__list-link--md--PaddingX);
     }
 
     // States
     &.pf-m-hover,
-    &:hover {
-      color: var(--pf-c-nav__list-link--hover--Color);
-      background-color: var(--pf-c-nav__list-link--hover--BackgroundColor);
-
-      &::after {
-        background-color: var(--pf-c-nav__list-link--hover--after--BackgroundColor);
-      }
-    }
-
+    &:hover,
     &.pf-m-active,
-    &:active {
-      font-weight: var(--pf-c-nav__list-link--active--FontWeight);
-      color: var(--pf-c-nav__list-link--active--Color);
-      background-color: var(--pf-c-nav__list-link--active--BackgroundColor);
-
-      &::after {
-        background-color: var(--pf-c-nav__list-link--active--after--BackgroundColor);
-      }
-    }
-
+    &:active,
     &.pf-m-focus,
     &:focus {
-      font-weight: var(--pf-c-nav__list-link--focus--FontWeight);
-      color: var(--pf-c-nav__list-link--focus--Color);
-      background-color: var(--pf-c-nav__list-link--focus--BackgroundColor);
+      --pf-c-nav__list-link--Color: var(--pf-c-nav__list-link--hover--Color);
+      --pf-c-nav__list-link--FontWeight: var(--pf-c-nav__list-link--m-current--FontWeight);
```

Comment:
this font weight is grouped in with `:hover` but should only be on `:focus` and `:active`

Reply:
updating this because the design doesnt show a hover state, but better to be consistent with the other navs